speechly-unity/Assets/Speechly/MicToSpeechly.cs

```diff
@@ -29,39 +29,49 @@ public static MicToSpeechly Instance
   [Tooltip("Capture device name or null for default")]
   public string CaptureDeviceName = null;
   public int MicSampleRate = 16000;
-  public int MicBufferLengthMillis = 1000;
   [Tooltip("Milliseconds of history data to send upon StartContext to capture lead of the utterance.")]
-  public int SendHistoryMillis = 200;
+  public int FrameMillis = 30;
+  [Range(1, 32)]
+  public int HistoryFrames = 5;
```

Comment:
This is a rather small number of history frames, which is good for the responsiveness, but might make it hard to tune the VAD for good accuracy. As long as this works, I'm fine with the parameters, just pointing out that this could be higher.

Reply:
I was thinking of the same, although it should match and in practice works rather well. There was one occasion with missing leading audio, so if you don't mind I'll add a separate setting for total history (which will be sent upon activation) and number of frames used for activation analysis.

Reply:
Added VADHistory to control the number of frames analyzed, and HistoryFrames which is the total history size.